.eslintrc.js

```diff
@@ -12,17 +12,20 @@ module.exports = {
         'json/sort-package-json': false,
     },
     rules: {
-        'import/no-unresolved': 'off',
+        'class-methods-use-this': 'off',
         'import/extensions': 'off',
+        'import/no-unresolved': 'off',
         'import/no-extraneous-dependencies': 'off',
-        'class-methods-use-this': 'off',
+        'no-shadow': 'off',
+        'no-use-before-define': 'off',
+        '@typescript-eslint/no-var-requires': 'off',
```

Comment:
These new lines were moved from pie-docs to root config. As a result, I've been able to delete the pie-docs eslintrc.

It also means that we can eventually remove the `// eslint-disable-line` comments we were using elsewhere, now it's applied globally.